### PR TITLE
docs: specify all ECE specific commands and add icons to .adoc files

### PR DESCRIPTION
**Pull request by karencfv**
## Description
This PR is separated in to two.

The first commit adds ECE specific text to all subcommands not just commands as it was before.

The second commit substitutes this text for `{ece-icon}` so our online documentation looks better with an icon instead of text.

@nrichers I'm assuming this will work fine? I looked at the kibana repo and didn't find any actual icon added to their repo

## Related Issues
Related: https://github.com/elastic/ecctl/issues/335

## Motivation and Context
Make the docs pretty 😊 

## Types of Changes

- [x] Documentation

### CODE REVIEW

File: cmd/auth/command.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ import (
 // Command is the auth subcommand
 var Command = &cobra.Command{
 	Use:     "auth",
-	Short:   "Manages the platform auth",
+	Short:   "Manages authentication",
```
Comment (marclop) [inline cmd/auth/command.go:29]: Maybe add `settings` at the end?
```suggestion
	Short:   "Manages authentication settings",
```

File: scripts/generate-docs.sh [hunk 2]
```diff
@@ -39,6 +39,9 @@ done
 # Strip extraneous .adoc out of section IDs and xrefs
 sed -i'.bak' -e 's/\.adoc//g' ecctl*.adoc
 
+# Replace ECE availability text for ece icon
+sed -i'.bak' -e 's/(Available for ECE only)/{ece-icon}/g' ecctl*.adoc
```
Comment (nrichers) [inline scripts/generate-docs.sh:43]: Kudos on doing this for the Asciidoc even through the Markdown doesn't really support it. ❤️

Comment (karencfv) [reply scripts/generate-docs.sh:43]: 🙇‍♀️

**Comment by karencfv** [comment]
@nrichers yeah, I think we could do that 🤔 unless @marclop has any objection :)

**Pull request merged**